src/rust/plugin-registry/src/db_client.rs

```diff
@@ -0,0 +1,78 @@
+use grapl_utils::future_ext::GraplFutureExt;
```

Comment:
The majority of the sqlx stuff in this file is just refactored out of previous code in server.rs.